central/sensor/service/connection/connection_impl.go

```diff
@@ -830,7 +830,7 @@ func (c *sensorConnection) ObjectsDeletedByReconciliation() (map[string]int, boo
 
 func (c *sensorConnection) CheckAutoUpgradeSupport() error {
 	if c.sensorHello.GetHelmManagedConfigInit() != nil && !c.sensorHello.GetHelmManagedConfigInit().GetNotHelmManaged() {
-		return errors.New("cluster is Helm-managed and does not support auto upgrades; use 'helm upgrade' or a Helm-aware CD pipeline for upgrades")
+		return errors.New("secured cluster version is managed by external tools (helm, operator)")
```

Comment:
```suggestion
		return errors.New("External tools (Helm or Operator) control the secured cluster version.")
```

Reply:
This message reads good, however our linter may not be happy about that, as error messages in Go should not start with capital letter and not end with a period. I will check whether we can add some exception here.